gradle/overrides.gradle

```diff
@@ -5,8 +5,9 @@ configurations.all {
         }
         preferProjectModules()
 
-        cacheDynamicVersionsFor 0, 'days'
-        cacheChangingModulesFor 0, 'days'
+        def dependencyCacheDays = Integer.valueOf(project.dependencyCacheDays)
+        cacheDynamicVersionsFor dependencyCacheDays, 'days'
+        cacheChangingModulesFor dependencyCacheDays, 'days'
```

Comment:
I think making this configurable is good, b/c downloading snapshots, even from maven central, seems to be flaky, especially on MacOS.